background/score/activities.go

```diff
@@ -105,15 +105,13 @@ func (s *ScoreUpdateWorker) RefreshLocationStateActivity(ctx context.Context, ac
 		}
 		// TODO: refactor
 		// this overrides the original score in the metric struct and it'll be saved back to database as autonomy score
-		autonomyScore, autonomyScoreYesterday, _ := score.CalculatePOIAutonomyScore(resourceMetric.Resources, metric)
-		metric.Score = autonomyScore
-		metric.ScoreYesterday = autonomyScoreYesterday
+		autonomyScore, _, autonomyScoreDelta := score.CalculatePOIAutonomyScore(resourceMetric.Resources, metric)
```

Comment:
Since we're going to save the score at the top level, you can just remove my comments at line 106-107.

Reply:
Deleted